lib/provider/vpc_data_types.go

```diff
@@ -42,6 +42,11 @@ type Profile struct {
 	CRN  string `json:"crn,omitempty"`
 }
 
+// Device ...
+type Device struct {
```

Comment:
This is not really required as we are returning path not the ID, so I think we can directly have DevicePath in VolumeAttachment Responce

Reply:
Will nextgen also returns path ?  i assume it will return "id" ,that needs to be searched on Host (/dev/disk/by-id/) . So let the Struct Model Object be "ID", its in sync with the riaas API Model Object.

Reply:
This struct is for library not the RIaaS so this will always contain DevicePath only

Reply:
please check my previous comments, I am saying not to have deferent  `Device` struct instead put `DevicePath` in  VolumeAttachment Response struct

Reply:
Lets keep the attachment object inline with riaas model..Lets not change the Structures too much, we might need additional info like "ID" in future part of device.